GAE/src/org/waterforpeople/mapping/app/web/CronCommanderServlet.java

```diff
@@ -65,9 +68,37 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
             generateNotifications();
         } else if ("purgeDeviceFileJobQueueRecords".equals(action)) {
             purgeDeviceFileJobQueueRecords();
+        } else if ("purgeReportRecords".equals(action)) {
+            purgeReportRecords();
         }
     }
 
+    /**
+     * scans for and deletes Report entries that are either more than one year old
+     */
+    private void purgeReportRecords() {
+        Calendar deadline = Calendar.getInstance();
+        deadline.add(Calendar.YEAR, ONE_YEAR_AGO);
+        log.info("Starting scan for Report entries, older than: " + deadline.getTime());
```

Comment:
Can we change this to debug as well?

Reply:
Sure. Though one log line per week will not kill us...